src/components/ExerciseForm.tsx

```diff
@@ -0,0 +1,121 @@
+import React,{ useState } from 'react'
+import exerciseService from "../services/exercises";
+import { useNavigate } from "react-router-dom";
+
+const ExerciseForm = () => {
+
+    const selectedValue = "Easy";
+    const [title, setTitle] = useState("");
+    const [complexity, setComplexity] = useState("");
+    const [textExercise, setTestExercise] = useState("");
+    const navigate = useNavigate();
+  
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+      e.preventDefault();
+  
+      exerciseService
+        .create(title, textExercise, complexity)
+        .then(() => {
+          navigate("/exercises");
+        })
+        .catch((error) => {
+          console.log(error);
+        });
+    };
```

Comment:
This function should be in each create/edit page and handleSubmit should come from props aswell